lib/DragNDrop.pm

```diff
@@ -159,18 +166,20 @@ sub HTML {
 sub TeX {
 	my $self = shift;
 
-	my $out = "\n\\hrule\n\\vspace{0.5\\baselineskip}\n";
+	my $out =
+		"\n\\hrule\n\\vspace{0.5\\baselineskip}\n\\newif\\ifdndcolumns\n"
+		. "\\ifdim\\linewidth<$self->{multicolsWidth}\\relax\\dndcolumnsfalse\\else\\dndcolumnstrue\\fi\n";
 
 	for my $i (0 .. $#{ $self->{defaultBuckets} }) {
 		my $bucket = $self->{defaultBuckets}[$i];
 		if ($i != 0) {
-			if   ($i % 2 == 0) { $out .= "\n\\hrule\n\\vspace{0.5\\baselineskip}\n" }
-			else               { $out .= "\n\\ifdefined\\nocolumns\\else\\hrule\n\\vspace{0.5\\baselineskip}\n\\fi" }
+			if   ($i % 2 == 0) { $out .= "\n\\hrule\n\\vspace{0.5\\baselineskip}\n"; }
+			else               { $out .= "\n\\ifdndcolumns\\else\\hrule\n\\vspace{0.5\\baselineskip}\\fi\n"; }
 		}
 
 		$out .=
 			"\n\\begin{minipage}{\\linewidth}\n\\setlength{\\columnseprule}{0.2pt}\n"
-			. "\\ifdefined\\nocolumns\\begin{multicols}{2}\\else\\fi\n"
+			. "\\ifdndcolumns\\begin{multicols}{2}\\else\\fi\n"
```

Comment:
The `\else` here is not needed. Same for two places below.

```suggestion
			. "\\ifdndcolumns\\begin{multicols}{2}\\fi\n"
```

Reply:
I'll remove them, they were just there before so I left them.

Reply:
Okay, removed the unneeded else statements.